### PR TITLE
Frontend

### PR DESCRIPTION
**Pull request by samirpdl**
- Grid View In react JS.

### CODE REVIEW

**Comment by mesaugat** [comment]
We need to opt out of `Bootstrap` and `jQuery`.

File: frontend/public/js/src/application.js [hunk 1]
```diff
@@ -0,0 +1,134 @@
+// NOTICE!! DO NOT USE ANY OF THIS JAVASCRIPT
```
[the remark this answers is not in the document]
Comment (samirpdl) [reply frontend/public/js/src/application.js]: @mesaugat it was used by Abash dai for building UI. And it's based on Bootstrap.

**Comment by samirpdl** [comment]
@mesaugat The UI is based on `Bootstrap` so not sure if we can opt it out.

**Comment by mesaugat** [comment]
Closed as we won't be using any libraries except for React.

**Pull request closed without merging**